node-admin/pom.xml

```diff
@@ -157,6 +157,23 @@
                     </compilerArgs>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>target/node-admin-app/components</outputDirectory>
+                            <includeArtifactIds>bcprov-jdk15on,bcpkix-jdk15on</includeArtifactIds>
```

Comment:
Must the artifactIds be hardcoded? It is easy to forget to update this list when adding new components.

Reply:
A dependency may already be provided as part of the standalone-container, so I think we need to verify each.